standard_forms/outbound-invoice-portrait-with-logo/latest/src/header.html

```diff
@@ -1,49 +1,75 @@
-{% macro dateOrNA(datetime) %}
-{% if datetime %}{{ datetime | date(format="%d/%m/%Y", timezone=arguments.timezone) }}{% else %}{{t(k="messages.not-applicable", f="N/A")}}{% endif %}
-{% endmacro input %}
+{% macro dateOrNA(datetime) %} {% if datetime %}{{ datetime |
+date(format="%d/%m/%Y", timezone=arguments.timezone) }}{% else
+%}{{t(k="messages.not-applicable", f="N/A")}}{% endif %} {% endmacro input %}
 
-<table class="header_image_section" style="width: 100%;height: 98%;">
-    <tr>
-        <th style="text-align:left;width:10%;height:70%;">
-            <img class="logo" src="{{ data.store.logo }}" />
-        </th>
-        <th style="text-align:left;width:20%;">
-            <span style="font-weight:bold;">{{ data.store.storeName }}</span><br>
-            <span style="font-size:6pt;">{{ data.store.name.address1 }}</span><br>
-            <span style="font-size:6pt;">{{ data.store.name.address2 }}</span><br>
-            <span style="font-size:6pt;">
-                {{t(k="label.phone",f="Telephone")}}: {{ data.store.name.phone }}
-            </span><br>
-            <span style="font-size:6pt;">
-                {{t(k="label.email",f="Email")}}: {{ data.store.name.email }}
-            </span>
-        </th>
-        <th style="text-align:right;width:70%;font-size:28pt;">
-            {{ t(k="label.outbound-shipment", f="Outbound Shipment") }}<br>
-        </th>
-    </tr>
+<!-- Main form has 7 columns, so we'll work with that here (means excel report header and 
+ data table are the same width) -->
+<table class="header_section">
```

Comment:
Yeah so main changes to consider are here really - will need to update form headers to use table rather than spans etc...

Added handling of row/col span so we can support merged cells in excel